### ISSUE TITLE
Local Library Download Problem

### ISSUE THREAD

**Issue by soiadmahedi** [Sketchware-Pro/Sketchware-Pro#1264]
**Describe the bug**
When i want download and add local library to this error is showing or Sketchware Application is crashed. 

**To Reproduce**
1. Open Any Project in Sketchware 
2. Go to Local Library 
3. Download this Library "androidx.draganddrop:draganddrop:1.0.0"
...

**Screenshots**
![Screenshot_2024-09-24-20-19-23-51_423e58f25c30a02453fa5d8c82fc37f2](https://github.com/user-attachments/assets/76802fb6-b8b5-41ea-86a6-bfa91e2237d8)
![Screenshot_2024-09-24-20-18-38-02_423e58f25c30a02453fa5d8c82fc37f2](https://github.com/user-attachments/assets/00cf9ddd-a540-4563-83ba-231737682fcc)

**Error is**

Sketchware Pro v6.4.0-SNAPSHOT-8f280f3-minApi26 (150)
base.apk size: 136 MB (135615978 B)
Locale: en_BD
SDK version: 34
Brand: realme
Manufacturer: realme
Model: RMX3710

```
java.io.UncheckedIOException: java.io.IOException: close failed: EIO (I/O error)
	at java.util.zip.ZipFile$CleanableResource.run(ZipFile.java:858)
	at jdk.internal.ref.CleanerImpl$PhantomCleanableRef.performCleanup(CleanerImpl.java:212)
	at jdk.internal.ref.PhantomCleanable.clean(PhantomCleanable.java:133)
	at java.lang.Daemons$FinalizerDaemon.doClean(Daemons.java:388)
	at java.lang.Daemons$FinalizerDaemon.processReference(Daemons.java:358)
	at java.lang.Daemons$FinalizerDaemon.runInternal(Daemons.java:335)
	at java.lang.Daemons$Daemon.run(Daemons.java:131)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.io.IOException: close failed: EIO (I/O error)
	at libcore.io.IoBridge.closeAndSignalBlockedThreads(IoBridge.java:313)
	at java.io.RandomAccessFile.close(RandomAccessFile.java:726)
	at java.util.zip.ZipFile$Source.close(ZipFile.java:1460)
	at java.util.zip.ZipFile$Source.release(ZipFile.java:1407)
	at java.util.zip.ZipFile$CleanableResource.run(ZipFile.java:849)
	... 7 more
Caused by: android.system.ErrnoException: close failed: EIO (I/O error)
	at libcore.io.Linux.close(Native Method)
	at libcore.io.ForwardingOs.close(ForwardingOs.java:176)
	at libcore.io.BlockGuardOs.close(BlockGuardOs.java:113)
	at libcore.io.ForwardingOs.close(ForwardingOs.java:176)
	at libcore.io.IoBridge.closeAndSignalBlockedThreads(IoBridge.java:311)
	... 11 more

```

**Comment by ilyassesalama** [comment]
I tried downloading the library you mentioned and it downloaded just fine. I used `v6.4.0-SNAPSHOT-96242e8e0-minApi26`.

**Comment by Mester-Dark** [comment]
Thank you

**Comment by RoyTheReal** [comment]
This is still an issue in newer snapshots

**Comment by Ohyeah66h** [comment]
Yeah this is so aggravating. Please some fix :/

**Comment by hasrat-ali** [comment]
Fixed with #1364

**Comment by oramirezd** [comment]
I am think not search in "maven central" .
